hack/dataplaneParser/iptable/iptable.go

```diff
@@ -0,0 +1,114 @@
+package iptable
+
+import (
+	"os"
+	"time"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+	"golang.org/x/sys/unix"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type Iptables struct {
+	Name   string
+	Chains map[string]*IptablesChain
+}
+
+type IptablesChain struct {
+	Name  string
+	Data  []byte
+	Rules []*IptablesRule
+}
+
+type IptablesRule struct {
+	Protocol string
+	Target   *Target
+	Modules  []*Module
+}
+
+type Module struct {
+	Verb           string
+	OptionValueMap map[string][]string
+}
+
+type Target struct {
+	Name           string
+	OptionValueMap map[string][]string
+}
+
+type Direction string
+
+func grabIptablesFileLock(f *os.File) error {
+	return unix.Flock(int(f.Fd()), unix.LOCK_EX|unix.LOCK_NB)
+}
+
+// grabs iptables v1.6 xtable lock
+func GrabIptablesLocks() (*os.File, error) {
```

Comment:
Do we need to re-write this function here ? I think we already have a grabiptableslock funciton in pkg iptm, we should probably use that.